dorf_tables.lua

```diff
@@ -1,52 +1,121 @@
 -- dorf_tables has job distribution configurations, random number information for attributes generation, job configurations, profession configurations, dwarf types(ie. attributes/characteristic) configurations
--- usage: loaded by pimp-it.lua
--- by josh cooper(cppcooper) [created: 12-2017 | last edited: 12-2018]
+-- usage: loaded by dwopit.lua
+-- by josh cooper(cppcooper) [created: Dec. 2017 | last modified: 2020-02-21]
 --@ module = true
 
+local utils=require('utils')
+local validArgs = utils.invert({
+ 'help',
+ 'list'
+})
+local args = utils.processArgs({...}, validArgs)
+
 local help = [====[
 
 dorf_tables
 ===========
-Data tables for pimp-it.lua.
-
-Usage: load inside secondary script (pimp-it.lua)
+Data tables for dwopit.lua.
+
+Arguments:
+    list [jobs|professions|types]
+
+Examples
+    # dorf_tables -list jobs
+    # dorf_tables -list professions
+    # dorf_tables -list types
+
+-~~~~~~~~~~
+The data tables defined are described below.
+
+job_distributions:
+    Defines thresholds for each column of distributions. The columns must
+    add up to the values in the thresholds row for that column.
+    Every other row references an entry in 'jobs'
+
+jobs:
+    Defines dwopit's nameable jobs. This is preferable to taking every
```

Comment:
If you wrap the script name in backticks, it will produce a link. For example (hopefully this works):
```suggestion
    Defines `dwarf-op`'s nameable jobs. This is preferable to taking every
```